public/maillog.php

```diff
@@ -0,0 +1,150 @@
+<?php
+/*
+* to do :   - add pagination
+*	          - sort by mounth
+* 	        - remove old logs
+*/
+
+require_once('common.php');
+
+authentication_require_role('admin');
+
+$CONF = Config::getInstance()->getAll();
+$smarty = PFASmarty::getInstance();
+
+$PALANG = $CONF['__LANG'];
```

Comment:
Do you really need `$CONF` and `$PALANG`? Smarty gets them on its own, and you don't seem to use them otherwise.

(Besides that, reading only the config option you need is preferred.)